build_projects/Microsoft.DotNet.Cli.Build.Framework/Enumerations/BuildArchitecture.cs

```diff
@@ -3,6 +3,7 @@ namespace Microsoft.DotNet.Cli.Build.Framework
     public enum BuildArchitecture
     {
         x86 = 1,
-        x64 = 2
+        x64 = 2,
+        arm = 3
```

Comment:
While you are augmenting this @jyoungyun, can you please add support for Arm64 as well?